### DISCUSSION TITLE
NICs Confirmed working

### DISCUSSION THREAD

**Discussion by donatengit** [donatengit/AppleIGB#6] [General]
Hi,

Please post here if your NIC/MB works with the driver stably and at full speed.

I'll start:

Ryzen 5 B450 Gigabyte [Aorus Pro-Wifi](https://www.gigabyte.com/Motherboard/B450-I-AORUS-PRO-WIFI-rev-10#kf), 100mb, 1GB

**Comment by donatengit** [comment]
b450 [MSI Pro Carbon](https://github.com/donatengit/AppleIGB/issues/5)

**Comment by Allubz** [comment]
ASUS X299 Deluxe /w Intel I211-AT
OC 0.8.1 / Monterey 12.3.1

Old SmallTree driver wouldn't properly connect. Without any driver the system would  crash right after the GUI loads, when the cable is connected. 

With this driver everything seems to work properly. Thanks!

**Comment by gabevf** [comment]
Works with X299 Designare EX... _BUT_ it has a few issues:

- does not connect right away (might need to unplug and plug the cable back in)
- seem to make the system unstable/slow/laggy

**Reply by donatengit** [reply]
This discussion is for working setups. Could you please create separate issue and provide `sudo dmesg | grep -i igb` there (right after the cable plug-in)? For unstable system could you please disable the driver in config.plist and confirm the system works ok without it? Thanks

**Reply by donatengit** [reply]
Hey @gabevf 
 
 Any news?

**Reply by gabevf** [reply]
Hi, sry for the delay, but I finally got around to testing it.
I think there might have been something in my EFI interfering with it, because it seems to work fine now.

here's what I got right after plugging in the cable, which seemed to work on the first try:

```
[  619.843089]: igb: setLinkUp() ===>
[  619.855016]: igb: setLinkUp() <===
```

Can't tell you for sure what was interfering, but my best guess is that it's either related to changes I made to TB3 SSDT's or AppleXcpmCFgLock quirk setting.

Hope this helps!

**Reply by Dawn13** [reply]
Works perfectly on Gigabyte Z390 Designare ! OC_0.8.2   Monterey 12.4
Intel 1211 0x1458/0xE000

Thanks to all involved !

**Comment by n374** [comment]
[GIGABYTE z270n-wifi](https://www.gigabyte.com/Motherboard/GA-Z270N-WIFI-rev-10/sp#sp), dual Intel® GbE LAN: `i211` and `i219`
with AppleIGB version `5.7.2-im-eee-stall`, `i211` works great.

OpenCore 0.8.2
Monterey 12.4 (21F79)

Thank you for your time and work!

**Comment by seven-of-eleven** [comment]
[Gigabyte Designaire z390](https://www.gigabyte.com/Motherboard/Z390-DESIGNARE-rev-10#kf), dual Intel GbE LAN: `i211` and `i219`
with AppleIGB version `5.7.2-im-eee-stall`, `i211` working great.
- OpenCore 0.8.2
- Monterey 12.4

Thank you!

**Reply by seven-of-eleven** [reply]
Still working on same motherboard with:
- OpenCore 0.8.3
- Monterey 12.5 & Ventura (beta 4)
Thanks!

**Comment by donatengit** [comment]
[Asus Prime x470 ](https://github.com/donatengit/AppleIGB/issues/8)

**Comment by donatengit** [comment]
[MSI X470 Gaming Pro Carbon](https://github.com/donatengit/AppleIGB/issues/9)

**Comment by jjlorenzo** [comment]
`6770K` with `Z170X-UD5` at `1GB`
It's a dual LAN board, I normally use the `Intel I219V2` with `IntelMausi.kext` but recently I started to use both.
In Monterey without this kext, the system crash right after I plug the cable, with the kext everything works fine, the speed is the same as the other.

**Comment by Lorys89** [comment]
Works In my gigabyte b450i aorus pro wifi + ryzen 5 2600x
macos ventura rc2

I211 pci8086,1539

**Reply by Lorys89** [reply]
@donatengit If you have discord you can write me Lorys89#7942
I have to talk to you. 
See you soon

**Comment by nolevuhao** [comment]
I'm update ventura , asus b450i with intel i211 it work but when I'm use photoshop or sometime use another app , I can't connect internet with safari or any app need internet
instead I'm  use wifi is connect still normal

**Reply by donatengit** [reply]
Hey @nolevuhao 
Thanks for testing. I haven't tested the driver on Ventura yet, will be able to upgrade and test in one of the nearest weekends

**Comment by donatengit** [comment]
This fork is being deprecated, please check [README](https://github.com/donatengit/AppleIGB) for better option of the driver.
